src/Pyz/Shared/PaymentTemplate/PaymentTemplateConstants.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Pyz\Shared\PaymentTemplate;
+
+interface PaymentTemplateConstants
+{
+    /**
+     * @var string
+     */
+    public const APP_IDENTIFIER = 'af10ba73-179c-4998-8b07-016d37ed974a';
```

Comment:
this is how it is done on Algolia
we aren't using this constant at the moment, but on Algolia it is used to set `emitter` field when messaging tenants to start export or to update search endpoint
this can still be done here to notify tenants about available payment methods and their endpoints
also having a preset application ID seems to be the way for us so I decided to do this already :)

Reply:
Updated